Doc/library/struct.rst

```diff
@@ -266,13 +266,6 @@ platform-dependent.
 +--------+--------------------------+--------------------+----------------+------------+
 | ``P``  | :c:expr:`void \*`        | integer            |                | \(5)       |
 +--------+--------------------------+--------------------+----------------+------------+
-
-Additionally, if IEC 60559 compatible complex arithmetic (Annex G of the
-C11 standard) is supported, the following format characters are available:
-
-+--------+--------------------------+--------------------+----------------+------------+
-| Format | C Type                   | Python type        | Standard size  | Notes      |
-+========+==========================+====================+================+============+
 | ``F``  | :c:expr:`float complex`  | complex            | 8              | \(10)      |
```

Comment:
IMHO, now that the support is unconditional, the "F" and "D" rows should be listed after the "e" / "f" / "d" rows, this way all the floating types are together.